File: exs/mundo_1/javascript/010.js
/*
Desafio 010

Problema: Crie um programa que leia quanto dinheiro uma pessoa tem na carteira e mostre quantos dólares ela pode comprar

Resolução do problema:
*/

const input = require('readline-sync').question

let qntDinheiro = Number(input("Quanto dinheiro você tem? R$"))
let valorDolar = 5.65

let qntDolar = (qntDinheiro / valorDolar)

console.log(`Você pode comprar U$${qntDolar.toFixed(2)}`)

File: exs/mundo_1/ruby/010.rb
=begin
Desafio 010

Problema: Crie um programa que leia quanto dinheiro uma pessoa tem
          na carteira e mostre quantos dólares ela pode comprar.

Resolução do problema:
=end

print"Quantos Reais você tem? "
n = gets.chomp.to_f
dolar = 0.19 #preço do dolar no dia 4/1/2021
puts"Com esse dinheiro, você consegue comprar #{n*dolar}US$."